test/app/index.test.js

```diff
@@ -352,14 +352,16 @@ describe('X-GOVUK Component Renderer', () => {
       .expect(500));
 
     it('should work with HMRC components', (done) => request(app)
-      .get('/example-usage/hmrc/green-button')
+      .get('/example-usage/hmrc/timeline')
```

Comment:
This test failed as it's dependent on content in hmrc/design-system that had changed. We should probably look at our test strategy on this at some point, as it's a little brittle.